AbletonMCP_Remote_Script/handlers/midi.py

```
@@ -0,0 +1,446 @@
+        # Remove all notes -- try Live 11+ API first, fall back to legacy
+        # Use clip.length + 1 to include notes starting exactly at clip.length
+        if hasattr(clip, 'remove_notes_extended'):
+            clip.remove_notes_extended(0, 128, 0, clip.length + 1)
+        else:
+            clip.remove_notes(0, 0, clip.length + 1, 128)
+
+        return {
+            "cleared": True,
+            "notes_removed": notes_count,
+            "clip_name": clip.name,
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)